src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java

```diff
@@ -69,4 +72,26 @@ public void should_be_able_to_delete_all_users_drafts_in_service() throws Except
         draftStoreClient.deleteAllUsersDrafts();
         assertThat(draftStoreClient.readDraftPage().data).isEmpty();
     }
+
+    @Test
+    public void should_return_UP_for_liveness_check() {
+        given()
+            .accept(APPLICATION_JSON_VALUE)
+            .when()
+            .get("http" + draftStoreUrl.substring("https".length()) + "/health/liveness")
```

Comment:
is this how the rest of the calls are made? normally we set RestAssured.some insecure setting
to allow invalid certs

Reply:
it was more about the protocol in front due to the health endpoint not being under port 443.

Reply:
it is under port 443 as well